main.cpp

```diff
@@ -74,11 +24,14 @@ void testPairToNumber(
 }
 
 int main() {
+
     testNumberToPair(4, TelCoColorCoder::WHITE, TelCoColorCoder::BROWN);
     testNumberToPair(5, TelCoColorCoder::WHITE, TelCoColorCoder::SLATE);
 
     testPairToNumber(TelCoColorCoder::BLACK, TelCoColorCoder::ORANGE, 12);
     testPairToNumber(TelCoColorCoder::VIOLET, TelCoColorCoder::SLATE, 25);
 
+    TelCoColorCoder::PrintColorCodeManual();
```

Comment:
The other `test...` functions above have asserts in them. So it's enough to know they passed.
But the output of `PrintColorCodeManual` is not guarded by asserts. So it requires manual verification of 25 lines of output every time. Can you think of any asserts for it?